Assignments/stu5/assignments.py

```diff
@@ -157,6 +161,51 @@ def vowel_counter(sentence):
             vowel = vowel+1
     return vowel
 
+def calculator():
+    num1 = input("Enter number 1: ")
+    num2 = input("Enter number 2: ")
+    operator = input("Enter operation (+, *, /, -): ")
+
+    add = int(num1) + int(num2)
```

Comment:
This is not efficient.  This code performs 4 calculations for every operation.  Let's re-write this so that only one calculation is performed.

Reply:
UPDATED